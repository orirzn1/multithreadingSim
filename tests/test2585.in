L10
S8
O6
N6

T0
I@0x00000000
SUBI $5, $5, 0x1
STORE $2, $0, 0x4
SUBI $0, $2, 0x4
ADD $4, $2, $6
STORE $3, $0, 0x2
ADD $1, $1, $5
SUBI $6, $3, 0x1
HALT $0

T1
I@0x00000000
SUB $4, $3, $2
LOAD $4, $1, $3
ADDI $4, $1, 0x2
LOAD $7, $0, $1
ADDI $4, $6, 0x3
SUBI $5, $0, 0x4
SUBI $0, $2, 0x2
STORE $0, $2, 0x2
ADDI $5, $1, 0x1
SUBI $0, $2, 0x3
HALT $0

T2
I@0x00000000
SUB $4, $4, $1
SUB $5, $7, $1
ADDI $3, $2, 0x4
SUB $6, $4, $0
ADDI $4, $5, 0x4
LOAD $2, $5, 0x2
HALT $0

T3
I@0x00000000
STORE $2, $6, 0x2
SUBI $6, $4, 0x2
STORE $0, $1, 0x3
ADD $4, $6, $5
ADDI $3, $6, 0x3
HALT $0

T4
I@0x00000000
ADD $6, $4, $3
LOAD $7, $6, 0x1
ADDI $7, $5, 0x3
LOAD $3, $3, 0x1
SUBI $5, $7, 0x3
ADD $7, $6, $5
ADD $3, $0, $3
STORE $7, $7, $3
HALT $0

T5
I@0x00000000
STORE $3, $3, $1
ADD $7, $2, $6
SUBI $3, $7, 0x4
ADD $2, $6, $2
ADDI $5, $0, 0x2
SUB $4, $4, $4
SUBI $7, $1, 0x4
ADDI $3, $6, 0x2
HALT $0

D@0x0
0x4
0xA
0x5
0x8
0x4
