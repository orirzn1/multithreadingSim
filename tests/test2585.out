
---- Blocked MT Simulation ----

Register file thread id 0:
	R0 = 0xFFFFFFFC	R1 = 0xFFFFFFFF	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0xFFFFFFFF	R6 = 0xFFFFFFFF	R7 = 0x0
Register file thread id 1:
	R0 = 0xFFFFFFFD	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x3	R5 = 0x1	R6 = 0x0	R7 = 0xFFFFFFFC
Register file thread id 2:
	R0 = 0x0	R1 = 0x0	R2 = 0x4	R3 = 0x4	R4 = 0x4	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 3:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x1	R4 = 0xFFFFFFFE	R5 = 0x0	R6 = 0xFFFFFFFE	R7 = 0x0
Register file thread id 4:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 5:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x2	R4 = 0x0	R5 = 0x2	R6 = 0x0	R7 = 0xFFFFFFFC
Blocked MT CPI for this program 3.040000

-----Finegrained MT Simulation -----

Register file thread id 0:
	R0 = 0xFFFFFFFC	R1 = 0xFFFFFFFF	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0xFFFFFFFF	R6 = 0xFFFFFFFF	R7 = 0x0
Register file thread id 1:
	R0 = 0xFFFFFFFD	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x3	R5 = 0x1	R6 = 0x0	R7 = 0xFFFFFFFC
Register file thread id 2:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x4	R4 = 0x4	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 3:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x1	R4 = 0xFFFFFFFE	R5 = 0x0	R6 = 0xFFFFFFFE	R7 = 0x0
Register file thread id 4:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0xFFFFFFFC	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 5:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x2	R4 = 0x0	R5 = 0x2	R6 = 0x0	R7 = 0xFFFFFFFC
Finegrained Multithreading CPI for this program 1.140000

